src/site/content/en/blog/asyncify/index.md

```diff
@@ -0,0 +1,516 @@
+the job last time and we want to avoid printing "A" twice - and instead it comes straight to the
+"rewinding" branch. Once it's reached, it restores all the stored locals, changes mode back to
+"normal" and continues the execution as if the code was never stopped in the first place.
+
+### Transformation costs
+
+Unfortunately, Asyncify transform isn't completely free, since it has to inject quite a bit of
+supporting code for storing and restoring all those locals, navigating the call stack under
+different modes and so on. It tries to modify only functions marked as asynchronous on the command
+line, as well as any of their potential callers, but that still might result in ~50% overhead to the
+uncompressed code size.
+
+{% Img src="image/9oK23mr86lhFOwKaoYZ4EySNFp02/Im4hmQOYRHFcsg8UTfTR.png", alt="A graph showing code size overhead for various benchmarks, from near-0% under fine-tuned conditions to over 100% in worst cases",
+width="800", height="494" %}
+
+It's not ideal, but in many cases acceptable when the alternative is not having the functionality
+altogether or having to make significant rewrites to the original code.
+
+Make sure to always enable optimizations for the final builds to avoid it going even higher. You can
+also check [Asyncify-specific optimization
+options](https://emscripten.org/docs/porting/asyncify.html#optimizing) to reduce the overhead by
+limiting transforms only to specified functions and/or only direct function calls. There is also a
+minor cost to runtime performance, but it's limited to the async calls themselves. However, compared
+to the cost of the actual work, it's usually negligible.
+
+## Real-world demos
+
+Now that we've looked at the simple examples, let's move on to more complicated scenarios.
+
+As mentioned in the beginning of the article, one of the storage options on the Web is an
+asynchronous [File System Access API](https://web.dev/file-system-access/). It provides access to a
+real host filesystem from a Web application.
+
+On the other hand, there is a de-facto standard—[WASI](https://github.com/WebAssembly/WASI) - for
+WebAssembly I/O in the console and the server-side. It was designed as a compilation target for
+system languages, and exposes all sorts of filesystem and other operations in a traditional
+synchronous form.
+
+What if we could map one to another? Then we could compile any application in any source language
+with any toolchain supporting the WASI target, and run it in a sandbox on the Web, while still
+allowing it to operate on real user files! With Asyncify, we can do just that.
+
+In this demo, I've compiled Rust [coreutils](https://github.com/RReverser/coreutils) crate with a
+few minor patches to WASI, passed via Asyncify transform and implemented asynchronous
+[bindings](https://github.com/GoogleChromeLabs/wasi-fs-access/blob/main/src/bindings.ts) from WASI
+to File System Access API on the JavaScript side. Once combined with xterm.js, this provides a
```

Comment:
What's xterm.js? You haven't mentioned it before.

Reply:
Please link to it. 

I call this a broken conceptual link. The fix is either to make it an actual link or remove it completely.